deps/fs.lua

```diff
@@ -577,3 +577,30 @@ end
 function fs.createReadStream(path, options)
   return fs.ReadStream:new(path, options)
 end
+function fs.appendFile(filename, data, callback)
+  callback = callback or function() end
+  local function write(fd, offset, buffer, callback)
+    local function onWrite(err, written)
+      if err then return fs.close(fd, function() callback(err) end) end
+      if written == #buffer then
+        fs.close(fd, callback)
+      else
+        offset = offset + written
```

Comment:
maybe do `buffer = buffer:sub(offset)` so that the `#buffer` above matches what we're trying to write?